packages/appservice-rest/src/Utilities/KuduServiceUtility.ts

```diff
@@ -196,4 +196,19 @@ export class KuduServiceUtility {
             deployer : 'GitHub'
         };
     }
+
+    public async deployWebAppImage(appName: string, images: string, isLinux: boolean) {
+        if (!isLinux) {
+            throw new Error("Windows Containerized web app or multi container support is not available for Publish profile auth scheme.");
+        } 
+        try {
+            console.log(`##[debug]Deploying image ${images} to the webapp ${appName}`);
+            let headers = {'LinuxFxVersion': images};
+            await this._webAppKuduService.imageDeploy(headers);
+        }
+        catch(error) {
+            core.error('Failed to deploy image to Container web app.');

```

Comment:
please check if the error is printed twice because of this.

Reply:
no, error will be printed only once in the catch block in main.